analyzer/emerald/incoming.go

```diff
@@ -188,7 +187,7 @@ func extractRound(sigContext signature.Context, b *block.Block, txrs []*sdkClien
 		}
 		blockTransactionData.Raw = cbor.Marshal(txr.Tx)
 		blockTransactionData.RelatedAccountAddresses = map[string]bool{}
-		tx, err := common.VerifyUtx(sigContext, &txr.Tx)
+		tx, err := common.OpenUtxNoVerify(&txr.Tx)
 		if err != nil {
 			logger.Error("error verifying tx", "tx_index", txIndex, "err", err)
```

Comment:
Nit: Might be useful to also have `txr.Tx.Hash()`; if we're handling this incorrectly and an external complaint comes in about data from tx such-and-such missing, they'll probably reference it by hash.

Reply:
oh yeah good idea

Reply:
ugh and I just realized this omits the round too